.pylintrc

```diff
@@ -2,16 +2,52 @@
 ignore-patterns=serialization.py,message.py,__main__.py,.*_pb2.py,launch.py
 
 [MESSAGES CONTROL]
-disable=C0103,C0201,C0330,C0301,C0302,W1202,W1203,W0511,W0107,R,W
-# Remove eventually
-# general R, W
-# In particular, resolve these and other important warnings:
+disable=C0103,C0201,C0330,C0301,C0302,W1202,W1203,W0511,W0107,W0105,W0621,W0235,W0703,W0212,W0706,W0108,W0622,W0613,W0201,W0223,W0221,W0611,W0612,W0222,W1505,W0106,R0902,R0913,R0914,R1720,R1705,R0801,R0904,R0903,R0911,R0912,R1721,R0901,R1718,R1723,R1704,R0916,R0201,R1714,R1702,R0123,R0915,R1710,R1703,R1716,R1711,R0205,R0401,R1722
+
+## Resolve these:
 # W0703: broad-except
 # W0212: protected-access, mostly resolved
 # W0706: try-except-raise
 # W0108: unnecessary-lambda
+# W0622: redefined-builtin
```

Comment:
Many of these should eventually be resolved; for now most are disabled; in particular as they affect skills as well.